Graphs/Table/index.js

```diff
@@ -175,7 +176,7 @@ const TableGraph = (props) => {
             updateScroll,
         } = props;
 
-        if (!isEmpty(displayColumn)) {
+        if (!isEmpty(displayColumn) || isAllColumnSelected === true) {
```

Comment:
`!isEmpty(displayColumn) || isAllColumnSelected` should suffice since isAllColumnSelected always has a boolean value